.github/workflows/pipeline.yml

```diff
@@ -26,3 +26,23 @@ jobs:
       - uses: actions/setup-go@v2
       # use go cli
       - run: go version
+      - run: go build *.go
+
+      - name: Install doctl
+        uses: digitalocean/action-doctl@v2
+        with:
+         token: ${{ secrets.DO_API_KEY }}
+
+      - name: Publish Image to Container Registry
+        uses: ripplr-io/docr-docker-publish@v1
+        with:
+          image_path: do-kevinrobayna-docker-registry/helloworld
+
+      - name: Provision Droplet and deploy container
+        run: doctl compute droplet create "ci_cd_do_example" --image docker-18-04 --size s-1vcpu-1gb --region lon1 --user-data-file deploy.sh --wait
+
+      # - name: Delete Droplet if there's one
+      #   run: doctl compute droplet delete "ci_cd_do_example"
+
+      - name: clean up docker registry
```

Comment:
this is to avoid reaching the limit size on our free registry https://www.digitalocean.com/docs/container-registry/how-to/clean-up-container-registry/